### PR TITLE
APEXCORE-347 Streamline japicmp maven plugin configuration

### PR DESCRIPTION
**Pull request by vrozov**
@tweise Please review

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
     <jersey.version>1.9</jersey.version>
     <!-- do not change jetty version as later versions have problems with DefaultServlet -->
     <jetty.version>8.1.10.v20130312</jetty.version>
+    <oldVersion.version>3.3.0-incubating</oldVersion.version>
```
Comment (tweise) [inline pom.xml]: Would japicmp.oldVersion be more appropriate?

Comment (vrozov) [inline pom.xml]: project.old.version? japicmp.oldVersion sounds related to japicmp version.

**Comment by tweise** [comment]
Looks good. How about waiting with this PR until semver option in the plugin is available?

**Comment by vrozov** [comment]
It is not clear when the next version of japicmp will be available. It will be good to have PR merged before the next core release goes out.

**Comment by tweise** [comment]
Agreed, we have some time till next minor release.

## 

sent from mobile

**Comment by vrozov** [comment]
@tweise Please review.

**Pull request merged**